### PR TITLE
Refresh Ray to v2.9.2

### PR DESCRIPTION
**Pull request by Archana-Shinde1**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,16 +8,18 @@ source:
...
     - patches/0004-Remove-all-dependencies-from-setup.py.patch
     - patches/0005-Ignore-warnings-in-event.cc-and-logging.cc.patch
     - patches/0006-Add-bazel-linkopts-libs.patch
+    - patches/0009-fixed-grpc-issue.patch
     - patches/0010-Use-rules_perl-which-is-fixed-for-ppc.patch
     - patches/0011-Fixed-compilation-error.patch
-    - patches/0012-Changed-min-bazel-version-to-5.3.patch
     - patches/0013-Added-patch-for-rules_python-on-ppc64le.patch    #[ppc64le]
-    - patches/0014-allow-builds-with-root-user.patch    #[ppc64le]
+#    - patches/0014-allow-builds-with-root-user.patch    #[ppc64le]
```
Comment (cdeepali) [inline recipe/meta.yaml:22]: Is this intentionally commented?

**Review by cdeepali — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (Archana-Shinde1) [reply recipe/meta.yaml:22]: yes, not needed

**Comment by alexlang74** [comment]
this still has the shm issue, right?

**Comment by Archana-Shinde1** [comment]
> this still has the shm issue, right?

shm size issue were seeing on ppc, on X86 ray built successfully.

**Pull request merged**